### ISSUE TITLE
include custom location/path in test result if specified

### ISSUE THREAD

**Issue by shoekstra** [inspec/inspec#1542]
If specifying a file name for the `mysql_conf` resource - or maybe any resource where you can specify a location instead of using the defaults as decided by inspec? - the location/path should be included in the test result output.

If you have a test with a custom file path:

```
describe mysql_conf('/tmp/my_rep.cnf') do
  its('sync_binlog') { should eq nil }
end
```

Current output (and should remain if using default location):

```
  MySQL Configuration
     ✔  sync_binlog should eq nil
```

Proposed output if custom config specified:

```
  MySQL Configuration (/tmp/my_rep.cnf)
     ✔  sync_binlog should eq nil
```

**Comment by adamleff** [comment]
This isn't something we can fix globally, but most resources do provide this additional context. The `mysql_conf` one simply does not, and it will be easy to fix. :)

**Comment by adamleff** [comment]
Not such an easy fix due to how we parse the resource string from the example string. See #1546.

**Comment by clintoncwolfe** [comment]
I believe this was fixed on #2464 ; please re-open if I am mistaken.  Thanks!
